src/Transpiler.ts

```diff
@@ -1513,7 +1517,10 @@ export abstract class LuaTranspiler {
 
         let result = "";
         const identifier = node.name as ts.Identifier;
-        const methodName = this.transpileIdentifier(identifier);
+        let methodName = this.transpileIdentifier(identifier);
+        if (methodName === "toString") {
+            methodName = "__tostring";
```

Comment:
I must be blind, but how does it know to escape when calling this method?

Reply:
I don't really understand the question. When calling tostring(table), the __tostring metamethod of the table's metatable is invoked.

Reply:
Aah gotcha